test/known_issues/test-fs-copyfile-respect-permissions.js

```diff
@@ -39,8 +40,12 @@ function beforeEach() {
 // Test promises API.
 {
   const { source, dest, check } = beforeEach();
-  assert.throws(async () => { await fs.promises.copyFile(source, dest); },
-                check);
+  (async () => {
+    await assert.rejects(
+      async () => { await fs.promises.copyFile(source, dest); },
```

Comment:
```suggestion
      fs.promises.copyFile(source, dest),
```

`assert.rejects()` also takes a promise directly as first argument.